test/unit/helpers/buyers/applications_helper_test.rb

```diff
@@ -1,66 +1,24 @@
+# frozen_string_literal: true
+
 require 'test_helper'
 
 class Buyers::ApplicationsHelperTest < ActionView::TestCase
 
-  test "services_contracted should return a json array of service ids" do
-    buyer = FactoryBot.create(:buyer_account)
-    service_ids = []
-    service_ids << FactoryBot.create(:service_contract, user_account: buyer).service.id
-    service_ids << FactoryBot.create(:service_contract, user_account: buyer).service.id
-    assert_same_elements service_ids, JSON.parse(services_contracted(buyer))
-  end
-
-  test "services_contracted should return a empty array" do
-    buyer = FactoryBot.create(:buyer_account)
-    assert_equal services_contracted(buyer), [].to_json
-  end
-
-  test "service_plan_contracted_for_service" do
-    buyer = FactoryBot.create(:buyer_account)
-    service_contract = FactoryBot.create(:service_contract, user_account: buyer)
-    buyer.reload
-
-    service_plan = buyer.bought_service_plans[0]
-
-    hash = {}
-    hash[service_plan.service.id] = {id: service_plan.id, name: service_plan.name}
-
-    assert_equal service_plan_contracted_for_service(buyer), hash.to_json
-  end
-
-  test "service_plan_contracted_for_service should return a empty hash" do
-    assert_equal service_plan_contracted_for_service(Account.providers.new), {}.to_json
+  test 'new_application_form_metadata requires a provider' do
+    assert_raise ArgumentError do
+      new_application_form_metadata
+    end
```

Comment:
Well this is not needed :) It is how Ruby works with keywords argument, you do not need to test Ruby